packages/amplify-codegen/src/commands/models.js

```diff
@@ -104,12 +105,11 @@ function loadSchema(apiResourcePath) {
     return fs.readFileSync(schemaFilePath, 'utf8');
   }
   if (fs.pathExistsSync(schemaDirectory) && fs.lstatSync(schemaDirectory).isDirectory()) {
-    return fs
-      .readdirSync(schemaDirectory)
-      .map(file => path.join(schemaDirectory, file))
-      .filter(file => file.endsWith('.graphql') && fs.lstatSync(file).isFile())
-      .map(file => fs.readFileSync(file, 'utf8'))
-      .join('\n');
+    // search recursively for graphql schema files inside `schema` directory
+    const schemas = glob.sync([
+      path.join(schemaDirectory, '**/*.graphql')
```

Comment:
There is a `build/` folder under the `schemaDirectory` for amplify project, which has the built graphql files. Those files are not the input for model-gen and should be excluded in the glob expression. I think an additional line should be applied in glob expression: `!${schemaDirectory/build/**/*.graphql`

Reply:
The `schemaDirectory` here points to `project/typescriptapp/amplify/backend/api/typescriptapp/schema` as specified in the modelgen prompt. `build/` folder is out of scope for this folder and glob expression as it is one level above `schema` folder.